### ISSUE TITLE
Transfer demo/checkpoint to the Microsoft organization

### ISSUE THREAD

**Issue by NielsRogge** [microsoft/TRELLIS#9]
Hello @JeffreyXiang and team 🤗

I'm Niels and work as part of the open-source team at Hugging Face. I discovered your work through AK's daily papers, the paper page is here: https://huggingface.co/papers/2412.01506. The paper page lets people discuss about your paper and lets them find artifacts about it (your models for instance), you can also claim the paper as yours which will show up on your public profile at HF.

I've got some suggestions to improve its discoverability on the hub :)

* I've opened a PR to improve the model card: https://huggingface.co/JeffreyXiang/TRELLIS-image-large/discussions/1
* I've opened a PR to link the demo to the paper page: https://huggingface.co/spaces/JeffreyXiang/TRELLIS/discussions/2

Besides that, would you be interested in transferring the model and demo to https://huggingface.co/microsoft? This could enhance its visibility, we could add a [collection](https://huggingface.co/docs/hub/en/collections) which groups the model, demo and paper page together.

Let me know if you need any help!

Cheers,
Niels

**Comment by YuDeng** [comment]
Hi @NielsRogge,

Thank you for your thoughtful suggestions and for taking the time to improve the visibility of our work. We are happy to proceed with merging the PR for text-to-3D trending at this time.

Thanks once again for your assistance!

Best,
Yu

**Comment by YuDeng** [comment]
Sure. Let us merge the PR to enable download counts!

For the checkpoint and demo, we'd like to temporarily keep them on our HF profile before considering a transfer to Microsoft at a later time. 

Thanks again!

**Comment by YuDeng** [comment]
Hello, @NielsRogge , it's been a while. We're currently considering moving the HF demos and checkpoints to the Microsoft directory. Could you help us with this process? Thanks a lot!

**Comment by NielsRogge** [comment]
Hi @YuDeng! To transfer a checkpoint, you can go in the settings of this model repository: https://huggingface.co/JeffreyXiang/TRELLIS-image-large (Assuming you have access to it), and if the username `JeffreyXiang` is also part of the `microsoft` org on the hub (and has contributor rights), you can click "tranfer repository".

**Comment by YuDeng** [comment]
Wow, thanks for the quick reply! BTW, I wonder if the history from the previous repo (e.g., likes) will be transferred along with it.

**Comment by NielsRogge** [comment]
Yes, everything is preserved when transferring :)

**Comment by YuDeng** [comment]
> Yes, everything is preserved when transferring :)

That's great. 

We found that JeffreyXiang does not have the permissions to transfer the repository to Microsoft. Do you know how can we add thr account as a member of the Microsoft organization on HF?

**Comment by NielsRogge** [comment]
Do you have any contacts who are admin of the `microsoft` org on the hub?

**Comment by YuDeng** [comment]
Yes, we have found the contact information for the admin. Thank you for your help, I will close this issue.

**Comment by NielsRogge** [comment]
Hi @YuDeng is there any update on this? People have reported to me that the TRELLIS Space is down

**Comment by YuDeng** [comment]
> Hi [@YuDeng](https://github.com/YuDeng) is there any update on this? People have reported to me that the TRELLIS Space is down

@NielsRogge, sorry for the late reply. TRELLIS Space is back online now.
